assets-src/styles/sass/30-base/_tables.scss

```diff
@@ -52,6 +54,8 @@ https://adrianroselli.com/2020/11/under-engineered-responsive-tables.html
 @media screen {
     .table-wrap {
         border: 1px solid $border-color;
+		margin-bottom: 1rem;
+		margin-top: 1rem;
         max-width: 100% !important;
```

Comment:
Same comment: code indentation needs fixing.

Reply:
Please re-check after pulling latest commits. I'm not sure if my default IDE indent spacing has changed between spaces and tabs.